src/common/quantile.cc

```diff
@@ -374,6 +374,38 @@ void AddCutPoint(typename SketchType::SummaryContainer const &summary, int max_b
   }
 }
 
+template <typename SketchType>
+void AddCutPointSecure(typename SketchType::SummaryContainer const &summary, int max_bin,
+                     HistogramCuts *cuts) {
+    // For secure vertical pipeline, we fill the cut values corresponding to empty columns
+    // with a vector of minimum value
```

Comment:
We are filling it with 0, not the minimum value. Please update the comment.

Reply:
thanks, updated